### ISSUE TITLE
Generate OFFMol molecule from OEMol is failing

### ISSUE THREAD

**Issue by nividic** [openforcefield/openforcefield#344]
Hi there, 

I was trying to generate an OFF molecule starting from an OpenEye molecule. I’m using the openforcefield v 0.3.0 installed from conda on Mac Osx. Unfortunately, I’m getting the following error:
```python
Traceback (most recent call last):
  File "stereo.py", line 9, in <module>
    molecule = Molecule(oemol, allow_undefined_stereo=True)
  File "/Users/gcalabro/local/miniconda3/envs/smirnoff_new/lib/python3.6/site-packages/openforcefield/topology/molecule.py", line 3433, in __init__
    super(Molecule, self).__init__(*args, **kwargs)
  File "/Users/gcalabro/local/miniconda3/envs/smirnoff_new/lib/python3.6/site-packages/openforcefield/topology/molecule.py", line 1517, in __init__
    result = toolkit_registry.call('from_object', other)
  File "/Users/gcalabro/local/miniconda3/envs/smirnoff_new/lib/python3.6/site-packages/openforcefield/utils/toolkits.py", line 2964, in call
    return method(*args, **kwargs)
  File "/Users/gcalabro/local/miniconda3/envs/smirnoff_new/lib/python3.6/site-packages/openforcefield/utils/toolkits.py", line 364, in from_object
    return self.from_openeye(object)
  File "/Users/gcalabro/local/miniconda3/envs/smirnoff_new/lib/python3.6/site-packages/openforcefield/utils/toolkits.py", line 749, in from_openeye
    raise UndefinedStereochemistryError(msg)
openforcefield.utils.toolkits.UndefinedStereochemistryError: (UndefinedStereochemistryError(...), 'Unable to make OFFMol from OEMol: OEMol has unspecified stereochemistry. oemol.GetTitle(): DrugBank_5354\nProblematic atoms are:\nAtom atomic num: 6, name: C16, idx: 15, aromatic: False, chiral: True with bonds:\nbond order: 1, chiral: False to atom atomic num: 6, name: C15, idx: 14, aromatic: False, chiral: False\nbond order: 1, chiral: False to atom atomic num: 15, name: P1, idx: 23, aromatic: False, chiral: False\nbond order: 1, chiral: False to atom atomic num: 16, name: S1, idx: 24, aromatic: False, chiral: False\nbond order: 1, chiral: False to atom atomic num: 1, name: H16, idx: 40, aromatic: False, chiral: False\n')
```
The script that I’m using is quite simple:

```python
from openeye import oechem
from openforcefield.topology import Molecule

oemol = oechem.OEMol()

with oechem.oemolistream("5354.oeb") as ifs:
    oechem.OEReadMolecule(ifs, oemol)

molecule = Molecule(oemol, allow_undefined_stereo=True)
```

The flag ```allow_undefined_stereo``` does not seem to have any effect. I have attached the zipped offending molecule as well.

[5354.oeb.zip](https://github.com/openforcefield/openforcefield/files/3258917/5354.oeb.zip)

**Comment by j-wags** [comment]
Thanks for the bug report, @nividic. 

This problem seems to be that the [Molecule constructor](https://github.com/openforcefield/openforcefield/blob/master/openforcefield/topology/molecule.py#L1517) doesn't pass `allow_undefined_stereochemistry` through to `ToolkitRegistry.from_object`.

I'll try to get a fix for this in before the next release.

**Comment by jchodera** [comment]
@nividic: Looks like @j-wags has fixed this in #351, but is there a reason you need to assign parameters to molecules with undefined stereochemistry? If you're simulating them, presumably you can only simulate one stereoisomer at a time, right? I'm just trying to understand why your use case requires passing around molecules with undefined stereochemistry, since we expect nobody would ever actually want to do this.
